sample-vagrant-host

```diff
@@ -0,0 +1,30 @@
+[master:vars]
```

Comment:
Can we use the cluster up example instead as we are already suggesting using cluster up in the README